docs/src/pages/components/use-media-query/UseWidth.js

```diff
@@ -3,21 +3,19 @@ import { ThemeProvider, useTheme } from '@material-ui/styles';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
 import { createMuiTheme } from '@material-ui/core/styles';
 
-/**
- * Be careful using this hook. It only works because the number of
```

Comment:
We use the same strategy in withWidth.js. It supports theme breakpoints customization. what's the motivation for using a different one? Should we use the same strategy everywhere?